### PR TITLE
Fix README.md links

### LINKED ISSUE #873: Fix README.md links
I broke a few links during the creation of the manual, they weren't really noticed until [now](https://amplipi.discourse.group/t/amplipro-streamer-first-impressions/320?u=amplipi)

Fix all links that now go to a 404 since I rearranged and ripped out all sorts of .md files

### PR DESCRIPTION
**Pull request by SteveMicroNova**
### What does this change intend to accomplish?
Closes #873
Replaces the destination of every broken/incorrect link in the primary README.md

### Checklist

* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] If applicable, have you updated the documentation/manual?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,15 +38,14 @@ In the future we plan to integrate with several home automation systems. Below i
...
 |openHAB|Yes|See https://next.openhab.org/addons/bindings/amplipi/|
 |Alexa|No|See [#25](http://github.com/micro-nova/AmpliPi/issues/25)|
 |Google Assistant|No|See [#69](http://github.com/micro-nova/AmpliPi/issues/69)|
 
-## Quick Start Guide
+## Manual
 
-If you are one of the lucky few to have a pre-release version of the AmpliPi,
-see [docs/QUICK_START.md](docs/QUICK_START.md) to get started.
+Find the device manual [here](amplipi.com/manual)
```
Comment (SteveMicroNova) [inline README.md:48]: There is no pre-release AmpliPi, we've deleted QUICK_START.md in favor of the manual

File: README.md [hunk 2]
```diff
@@ -60,11 +59,11 @@ Check out our open source wall panel [here](https://github.com/micro-nova/AmpliP
 
 ## Speakers
 
-Notes on picking and installing speakers for whole house audio can be found [here](docs/SPEAKERS.md).
+Notes on picking and installing speakers for whole house audio can be found [here](https://amplipi.discourse.group/t/tested-recommended-speakers-and-speaker-wire/31).
```
Comment (SteveMicroNova) [inline README.md:62]: This SPEAKERS.md file still exists, but was literally just a link to the discourse post

File: README.md [hunk 3]
```diff
@@ -38,15 +38,14 @@ In the future we plan to integrate with several home automation systems. Below i
 
 |Integrations|Supported|Notes|
 |--|--:|--|
-|Home Assistant|Yes|See https://github.com/brianhealey/hacs_amplipi|
+|Home Assistant|Yes|See https://github.com/micro-nova/hacs_amplipi|
```
[the remark this answers is not in the document]
Comment (linknum23) [reply README.md:41]: Eventually we could link to a markdown version of our manual. I know that doesn't exist in a useful state yet.

File: README.md [hunk 4]
```diff
@@ -60,11 +59,11 @@ Check out our open source wall panel [here](https://github.com/micro-nova/AmpliP
 
 ## Speakers
 
-Notes on picking and installing speakers for whole house audio can be found [here](docs/SPEAKERS.md).
+Notes on picking and installing speakers for whole house audio can be found [here](https://amplipi.discourse.group/t/tested-recommended-speakers-and-speaker-wire/31).
 
 ## Developing
 
-For details on how to help out with new features and bug fixes, check out [docs/DEVELOPING.md](docs/DEVELOPING.md).
+For details on how to help out with new features and bug fixes, check out [docs/DEVELOPING.md](docs/manual/DEVELOPING.md).
```
Comment (linknum23) [inline README.md:66]: Does developing belong in the manual directory?

**Review by linknum23 — COMMENTED** [review]
This gets us lots more functional links on the main page. Are there any other files we should check while we are at it?

**Comment by SteveMicroNova** [comment]
> This gets us lots more functional links on the main page. Are there any other files we should check while we are at it?

Not that I could find by looking through all the links in every .md file I could find

> Something that might check for stuff like this in the future: https://github.com/UmbrellaDocs/action-linkspector?tab=readme-ov-file

I am not someone who will be setting up github actions, I'll leave that to those who have a feel for what that actually entails

**Review by linknum23 — APPROVED** [review]

**Pull request merged**